packages/cli/src/__tests__/commands/Validate.test.ts

```diff
@@ -39,9 +39,9 @@ describe('validate', () => {
     // stderr
     expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
-                              Prisma schema warning:
-                              - Preview feature "nativeTypes" is deprecated. The functionality can be used without specifying it as a preview feature.
-                  `)
+                                    Prisma schema warning:
```

Comment:
Do you know why the indent is slightly changed?

Reply:
It's just `jest` with the `-u` flag, it's almost never consistent although the warnings didn't change at all